refs/schemas/priv/dictionary/actions/en.actions.json

```diff
@@ -0,0 +1,23 @@
+{
+  "ACCESS": "To access the data the System has on me",
+  "DELETE": "To have my data deleted",
+  "MODIFY": "To modify the existing or complement the incomplete data the System has on me",
+  "OBJECT": "To contact the user to offer products, services, or other promotions",
+  "PORTABILITY": "To take my day and ahve it transfered somewhere else",
+  "RESTRICT": "To restrict processing of my data",
+  "REVOKE-CONSENT": "To revoke prviousely given consent for data processing",
+  "TRANSPARENCY": "To demand information related to data processing practices and know if the system has data on me",
+  "TRANSPARENCY.DATA-CATEGORIES": "To know the categories of processing being done on the data the System has on me",
+  "TRANSPARENCY.DPO": "To know the contact details of the data protection officer",
+  "TRANSPARENCY.KNOWN": "To know if the System has data on me",
+  "TRANSPARENCY.LEGAL-BASES": "To know the legal bases for processing my data (including legitimate interests)",
+  "TRANSPARENCY.ORGANISATION": "To know the identity and the contact details of the organisation processing my data",
+  "TRANSPARENCY.POLICY": "To know the policies being applied to processing of data concerning me",
+  "TRANSPARENCY.PROCESSING-CATEGORIES": "To know the categories of processing being done on the data the System has on me",
```

Comment:
I think this is not the right definition, it is the same definition as TRANSPARENCY.PROCESSING-CATEGORIES
Might be "To know the categories of the data the system has on me" ?

Reply:
There exist two distinct things: 
- categories of data (like contact data, health data etc)
- categories of processing (collecting, storing, sharing).

I don't have a good view (I'm on mobile interface) of what you are point out to here. It might be that I copy-pasted a definition that needs changing.